discovery/consul/consul.go

```diff
@@ -95,9 +97,23 @@ type SDConfig struct {
 	Scheme       string             `yaml:"scheme,omitempty"`
 	Username     string             `yaml:"username,omitempty"`
 	Password     config_util.Secret `yaml:"password,omitempty"`
+	// See https://www.consul.io/docs/internals/consensus.html#consistency-modes,
+	// stale reads are a lot cheaper and are a necessity if you have >5k targets.
+	AllowStale bool `yaml:"allow_stale,omitempty"`
+	// By default use blocking queries () but allow users to delay
```

Comment:
Might it make sense to always use this with a value like 15s? Internally we already throttle all SD to 5s.

What is the () for?

Reply:
* Set default to 15s
* The `()` was to put a link to the doc